### DISCUSSION TITLE
Welcome to OCTA-Net-OCTA-Vessel-Segmentation-Network Discussions!

### DISCUSSION THREAD

**Discussion by YuhuiMa** [iMED-Lab/OCTA-Net-OCTA-Vessel-Segmentation-Network#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fpradelli94** [comment]
Hi! I'm a Ph.D. student working with retinal OCTA images and I came across your network. I think your network might be very useful for the research I am carrying  out right now, but I did not get how to use your model on images other those of the RITE dataset.

Is it possible to use the pre-trained model? In that case, can you share instructions on how to apply the model on a given image?

Thank you and best regards.
